### ISSUE TITLE
Can't get it to work

### ISSUE THREAD

**Issue by ExtremeCoolDude** [k-kirill/anki-libunity#4]
I can't get it to work. I use Papirus icons, but I tried on default icons and it doesnot work on default icons too. Most of the time, there is no error message, it simply doesnot work, sometimes it display's error message like this:

Anki 2.1.35 (84dcaa86) Python 3.8.1 Qt 5.15.1 PyQt 5.15.1
Platform: Linux
Flags: frz=True ao=True sv=1
Add-ons, last update check: 2020-11-16 03:05:39
Add-ons possibly involved: ⁨Anki libunity Linux only⁩

Traceback (most recent call last):
  File "aqt/main.py", line 520, in loadCollection
  File "aqt/gui_hooks.py", line 819, in __call__
  File "/home/red/.local/share/Anki2/addons21/901237400/__init__.py", line 23, in <lambda>
    gui_hooks.collection_did_load.append(lambda _: set_progress())
  File "/home/red/.local/share/Anki2/addons21/901237400/__init__.py", line 9, in set_progress
    progress = studied_card_count / (studied_card_count + due_card_count)
ZeroDivisionError: division by zero

**Comment by k-kirill** [comment]
Could you please clarify what is the panels/dock you use?
P.S. That error can happen if you load a collection with zero due cards. Do you use several collections?

**Comment by ExtremeCoolDude** [comment]
I'm currently using gnome with dash to dock extension. Gnome Shell 3.38.1

Thanks, the error happens when I have zero due cards.

**Comment by k-kirill** [comment]
I've fixed that error and uploaded the new version of the add-on.
As to the lack of the badge and progress, do you see them for other applications like Telegram desktop?

**Comment by k-kirill** [comment]
Looks like the issue is the desktop filename for Anki, for Flatpak it is `net.ankiweb.Anki.desktop`, but yours must be different. I've added the ability to change the application name in add-on's config and uploaded the new version of the add-on.

**Comment by ExtremeCoolDude** [comment]
Just saw this, thank you for the reply. Yup, the telegram icon shows messages :

![image](https://user-images.githubusercontent.com/50034073/104855558-8d147900-5935-11eb-8d7b-2cc485525c1a.png)

For anki atm its like this : 

![image](https://user-images.githubusercontent.com/50034073/104855582-a9181a80-5935-11eb-9a2d-d7ac7a49aa35.png)

updated to the latest verison of add on.
I did not install from Flatpak, downloaded anki directly from the website and installed it from the extracted folder.

I changed the app name to "anki"  on the config and holy shit it worked ! 

![image](https://user-images.githubusercontent.com/50034073/104855649-014f1c80-5936-11eb-9805-51104c375521.png)

Thank you!
